### DISCUSSION TITLE
cn/2024/12/10/idk-world/

### DISCUSSION THREAD

**Discussion by giscus** [hongtaoh/hongtaoh.github.io#194] [Announcements]
# cn/2024/12/10/idk-world/

https://hongtaoh.com/cn/2024/12/10/idk-world/

**Comment by Liam-Zhong** [comment]
专业的事情交给专业的人，你我都不是政治局常委，有些事就不用关心了。坐井观天也可以看到很好看的天。

**Reply by hongtaoh** [reply]
是的，哈哈。管好自己就可以了。

**Comment by weifanz** [comment]
关于变化：或许比起世界怎么变化，“希望世界是怎样的”更重要。
关于政治：一个平民能做的最大的事，是保护自己所珍视的人与生活。（这是最近韩国青年们让我明白的）
关于生命：或许存在本身就是意义。

**Reply by hongtaoh** [reply]
嗯嗯，有道理。

**Comment by BigCoke233** [comment]
关于生命的意义，我的思考是「留下痕迹」，人都有自我外化的需求，期待自己的行动能改变世界，从婴儿时期就开始了。倘若有目标和理想，其实就很容易想清楚这个「痕迹」是什么。

另外，我认为逻辑和理性不是一切的答案，过度思考反而容易陷入一种虚无感，理解真理反而需要更真实的感受和体会。

**Reply by hongtaoh** [reply]
有道理。你的博客真漂亮。

「留下痕迹」和我最近刚读完的《认知觉醒》里讲到的创造貌似有关联。忘了谁说的，幸福感两个来源之一是「影响力」，这个应该也和你说的有关系。

>另外，我认为逻辑和理性不是一切的答案，过度思考反而容易陷入一种虚无感，理解真理反而需要更真实的感受和体会。

是的。我是在搜索比特币是感觉很多事情不是逻辑能弄懂的，也没有对错之分。

**Comment by ZhangShenqing** [comment]
「知识是无知之海表面漂浮的诱饵」。

我想现代性溢出的年代里，人们最大的妄想就是「理性崇拜」，而忽视了「理性」本身的边界。人能地球的全貌，对于宇宙亦不过尘埃之于蚍蜉而已。生命的「意义」本身似乎注定不可被我们认知，大概唯一可以被人赋予的「意义」就是活着，过生活，经历，死的时候知道自己活过。
